clients/flash/air-client/src/org/bigbluebutton/air/main/views/UserInactivityView.as

```diff
@@ -1,40 +1,39 @@
-package org.bigbluebutton.air.main.views
-{
+package org.bigbluebutton.air.main.views {
 	import spark.components.Button;
-	import spark.components.Label;
+	import spark.layouts.HorizontalAlign;
+	import spark.layouts.HorizontalLayout;
 	import spark.layouts.VerticalLayout;
 	
-	import org.bigbluebutton.air.common.views.NoTabView;
-
-	public class UserInactivityView extends NoTabView
-	{
-		public var okButton:Button;
+	public class UserInactivityView extends MobilePopUp {

```

Comment:
I think this class should be renamed to UserInactivityPopUp as its base class is no longer a View.